README.md

```diff
@@ -99,23 +98,30 @@ cmake --build . --target install
 ```
 TODO: Add simple client and server source code
 
-### Example Programs
-If the `-DBUILD_EXAMPLES=ON` CMake option is used during building,
-example client and server executables will be built and placed
-in the `${CMAKE_BINARY_DIR}/bin` directory.
-Example configuration data is also provided in the `examples/data`
-directory.
+### Running a Test Client and Server Handshake
+If the `-DBUILD_TOOL=ON` CMake option is used during building,
+the XTT tool will be built and placed in the `${CMAKE_BINARY_DIR}/tool` directory.
+
+To create configuration data, run:  
+`./xtt genkey -v root_priv.bin -b root_pub.bin` to create a root key pair.  
+`./xtt genrootcert` to create a root certificate.  
+`./xtt genkey -v server_priv.bin -b server_pub.bin` to create a server key pair.  
+`./xtt genservercert` to create a server certificate.  
+
+Some example configuration data is also provided in the `examples/server`
+and `examples/client` directory.
 
 #### Server
 To run the example server, first copy the necessary example data
 into the working directory:
 ```bash
-cp ${xtt_root_directory}/examples/data/server/* .
+cp ${xtt_root_directory}/examples/server/* .
 ```
```

Comment:
Should we add a line here saying that we need to run the above-mentioned commands to generate the rest of the server configuration data?

Reply:
Yes either that or I was thinking maybe just make one section for creating input files. Putting the copying commands right above the tool commands, which one do you think is better?

Reply:
Hmm, maybe the best thing is to organize the `Usage` section into subsections: `Creating a Root`, `Provisioning a Server`, `Running a Test Server`, `Running a Test Client`. And in the `Running` subsections, use the CLI parameters to explicitly point to the correct locations of the files (to make it clear which files are required). My hope is that the up-coming changes to consolidate many of those files should keep those parameters from getting unwieldy.

My feeling is that being more explicit is better, to make sure a potential new user understands exactly how to run a demo.